packages/shared/src/utils.ts

```diff
@@ -43,4 +43,5 @@ export const mergeClasses = <ClassType = string | undefined | null>(...classes:
     .filter((className, index, array) => {
       return Boolean(className) && array.indexOf(className) === index;
     })
-    .join(' ');
+    .join(' ')
+    .trim();
```

Comment:
What about doing `return Boolean(className.trim()) && array.indexOf(className) === index;` instead?

That way you don't get something weird if you do `mergeClasses('lucide', ' ', 'lucide-circle')`.

Not sure about the performance impact tho.

Reply:
Hmm good idea, I will add that one too! Performance is negligible.